src/dream.mli

```diff
@@ -1387,7 +1387,14 @@ val session_label : request -> string
 val session_expires_at : request -> float
 (** Time at which the session will expire. *)
 
+(** {1 Flash Messages} *)
 
+val flash_messages : ?lifetime:float -> middleware
+type level = Debug | Info | Success | Warning | Error
```

Comment:
Is `Success` necessary, as separate from `Info`?

Reply:
Not really, I picked these names because they were the same as the ones used by Django. I'll delete it.